guides/users-guide/quickstart.md

```
@@ -10,6 +10,17 @@ icon: sign-posts-wrench
 2. Run `docker compose up` to start the application
 3. Now the application is running at `http://localhost`
 
+##### Optional Config:
```

Markdownlint

Expected: h3; Actual: h5
Heading levels should only increment by one level at a time

(MD001, heading-increment)